graylog2-server/src/main/java/org/graylog2/shared/rest/resources/system/RemoteDeflectorResource.java

```diff
@@ -20,6 +20,6 @@
 import retrofit2.http.POST;
 
 public interface RemoteDeflectorResource {
-    @POST("/system/deflector/cycle")
+    @POST("system/deflector/cycle")
```

Comment:
Yes, these are necessary.

Otherwise the custom context path would be overridden (because the paths in these Retrofit resource classes are absolute and not relative as they should be). I haven't looked into it too deeply but I haven't found a suitable mechanism in [Retrofit's documentation](https://square.github.io/retrofit/) yet.

Also see [Retrofit 2 — Url Handling, Resolution and Parsing](https://futurestud.io/blog/retrofit-2-url-handling-resolution-and-parsing).